### PR TITLE
Add basic distro for ronin

### PR DESCRIPTION
**Pull request by autra**
A basic distro for ronin. No fancy wallpapers, because I have already fought too much with the wallpapers :-) 

@peppsac please r?

### CODE REVIEW

File: distros/ronin/apps.list [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apps/bluetooth
+apps/bookmark
+apps/calendar
+apps/camera
+apps/clock
+apps/default_theme
+apps/download
+apps/email
+apps/findmydevice
```
Comment (peppsac) [inline distros/ronin/apps.list]: We could remove findmydevice I guess?

File: distros/ronin/apps.list [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+apps/fl
+apps/ftu
+apps/gallery
+apps/homescreen
+apps/keyboard
+apps/marketplace.firefox.com
+apps/music
+apps/network-alerts
+apps/operatorvariant
```
Comment (peppsac) [inline distros/ronin/apps.list]: Can we easily get rid of this?

File: distros/ronin/apps.list [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+apps/pdfjs
+apps/ringtones
```
Comment (peppsac) [inline distros/ronin/apps.list]: Can we easily get rid of this as well?

**Comment by peppsac** [comment]
Are the file added in `distros/ronin/wallpapers/Illus_Spark_Void.jpg` used?
If yes -> we should probably build landscape versions.
If not -> let's remove them for now (unless they're required somehow?)

**Comment by autra** [comment]
Yes, they appear in the list of wallpaper in settings (in the wallpaper app to be exact). But yeah, we'll put it back once we have correct sized wallpapers.

File: distros/ronin/apps.list (hunk 2 above)
Comment (autra) [inline distros/ronin/apps.list]: I guess so.

File: distros/ronin/apps.list (hunk 1 above)
Comment (autra) [inline distros/ronin/apps.list]: yup.

File: distros/ronin/apps.list (hunk 3 above)
Comment (autra) [inline distros/ronin/apps.list]: I'm going to test. I don't know if it is used only for ringtones, or also for other sounds.

**Comment by peppsac** [comment]
LGTM, merging

**Pull request merged**